packages/server/src/automations/steps/loop.js

```diff
@@ -43,7 +43,7 @@ exports.definition = {
           descriptions: "The amount of times the block ran",
         },
       },
-      required: ["success"],
+      required: ["success, items, iterations"],
```

Comment:
Minor issue here, this should be `["success", "items", "iterations"]` - right now its just an array of one string.

Reply:
Whoops! Fixed!